src/main/webapp/index.html

```diff
@@ -70,4 +61,7 @@
       </div>
     </nav>
   </body>
+  
+  <!-- JavaScript -->
```

Comment:
maybe write a comment explaining why you put the script tag at the bottom

Reply:
I don't think a link is necessary because it's also pretty standard. I've worked on projects before where all scripts are included in the bottom. Also, according to research it tends to be faster, so some people favor it: https://stackoverflow.com/a/143563.

The reason I did it was because I needed all elements to load before using the script file, so I can still add it if you think it's worth mentioning.